libs/partners/openai/langchain_openai/chat_models/base.py

```diff
@@ -897,7 +902,16 @@ def get_num_tokens_from_messages(self, messages: List[BaseMessage]) -> int:
         counting.
 
         OpenAI reference: https://github.com/openai/openai-cookbook/blob/
-        main/examples/How_to_format_inputs_to_ChatGPT_models.ipynb"""
+        main/examples/How_to_format_inputs_to_ChatGPT_models.ipynb
+
+        Args:
+            messages: The message inputs to tokenize.
+            tools: If provided, sequence of dict, BaseModel, function, or BaseTools
+                to be converted to tool schemas.
+        """
+        # TODO: Count bound tools as part of input.
+        if tools is not None:
+            warnings.warn("Counting tokens in tool schemas is not yet supported.")
```

Comment:
could we make it explicit that we're ignore tools
```suggestion
            warnings.warn("Counting tokens in tool schemas is not yet supported. Ignoring tools.")
```